src/app.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="en" style="background-color: black !important">
+<html lang="en" class="bg-secondary">
```

Comment:
Revert this. HTML should not have any class

Reply:
is that so? whys that? FOUC issue?

Reply:
Resolved on https://github.com/baseplate-admin/coreproject-v4-ui/pull/125/commits/cfbf81337bd7d69da514bdbb7ac5380e9dfaa6a8